src/garage/tf/misc/tensor_utils.py

```diff
@@ -13,6 +13,22 @@ def run(*input_vals):
     return run
 
 
+def broadcast_with_batch(param, input_var, batch_dim=1):
+    """
+    Broadcast an existing variable to match input batch dimension.
+
+    Args:
+        param (tf.Tensor): Variable to broadcast.
+        input_var (tf.Tensor): Input batch variable.
+        batch_dim (int): Define the first `batch_dim` dimension as
+            Batch dimension. If None, default to be 1.
+    """
+    assert batch_dim > 0
```

Comment:
use an if statement and raise a ValueError. asserts are not for validating user input.